### ISSUE TITLE
Mobile optimization

### ISSUE THREAD

**Issue by ardacebi** [Extum/material#24]
Mobile optimization should be supported that is being caused by shadows and cards that are added with the latest commits.

#### Individual issues
- #48
- #49
- #50
- #51

**Comment by ghost** [comment]
Mobile Theme Support will be rolling out within the next few weeks out of time.

**Comment by mikejones3** [comment]
Looks like you already have a badge not showing correctly but here is a screenshot to add to it:

https://imgur.com/gallery/ZV1rMI7

**Comment by ardacebi** [comment]
@mikejones3 Yes, I'm aware of it. I tried fixing it but my fix resulted in breaking the desktop version, so I switched back. 

I'm now working on switching to MDC so I'm busy, if you know a fix to it, please feel free to open a PR.

You will also be a contributor :)

**Comment by ghost** [comment]
The badge issue will be fixed in `1.0.0-beta.5`

**Comment by hammerhai** [comment]
Issues being closed since this is an incompatible version. A new one will be created if it reoccurs in 0.1.0-beta.12.

Details:
```
#0.1.0-beta.7=>24

This should not become an issue as the grid is provided by Flarum. Components are expected to stay within boundaries of grid.
```
